src/main/java/mate/academy/spring/model/User.java

```diff
@@ -1,12 +1,18 @@
 package mate.academy.spring.model;
 
+import java.util.Set;
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
 import javax.persistence.Table;
+import lombok.Data;
 
+@Data
```

Comment:
Using Lombok's `@Data` on Entity tables are very dangerous
see this article to find out why and how to fix it
https://thorben-janssen.com/lombok-hibernate-how-to-avoid-common-pitfalls/
https://habr.com/ru/company/haulmont/blog/564682/

Reply:
understood